### DISCUSSION TITLE
Atalhos de acessibilidade no topo do portal

### DISCUSSION THREAD

**Discussion by ortegavan** [techexpertspro/portal-ascepa#51] [General]
Acham que seria uma boa termos estes atalhos de acessibilidade no portal? Pergunto para prever no layout no Figma =)
<img width="1440" alt="Captura de Tela 2024-06-11 às 18 52 21" src="https://github.com/techexpertspro/portal-ascepa/assets/30645960/0ecb59bb-3957-4785-8cb7-7c798839a3a6">

**Comment by andrewarosario** [comment]
Acho que a melhor pessoa pra responder é o Leandro da ONG. Bora levar essa questão pra ele 😉

**Comment by paoloalmeida** [comment]
Boa Van!
Também não tenho essa resposta clara, pois nunca usei uma barra de atalho assim... Mas concordo com o Andrew em levarmos pra discussão com Leandro e Antonio.
